### PR TITLE
[16.0][MIG] shopinvader_lead: Migration to 16.0

### PR DESCRIPTION
**Pull request by matthieusaison**
need this PR to pass test:
https://github.com/OCA/rest-framework/pull/373

### CODE REVIEW

File: shopinvader_api_lead/schemas.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    city: str | None = None
+    phone: str | None = None
+    mobile: str | None = None
+    state_id: int | None = None
+    country_id: int | None = None
+    team_id: int | None = None
+    contact_name: str | None = None
+
+
+    # # faire un module supp pour ces champs
```
Comment (sebastienbeau) [inline shopinvader_api_lead/schemas.py:20]: TODO add a module: shopinvader_api_lead_firstname

File: shopinvader_api_lead/tests/test_shopinvader_lead.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        )
+
+        lead = self.env["crm.lead"].search([], order="id desc", limit=1)[0]
+        for key in check_data:
+            if isinstance(lead[key], models.Model):
+                self.assertEqual(lead[key].id, check_data[key])
+            else:
+                self.assertEqual(lead[key], check_data[key])
+        # TODO split shopinvader tests.test_notification
+        # self._check_notification("lead_confirmation", lead)
```
Comment (sebastienbeau) [inline shopinvader_api_lead/tests/test_shopinvader_lead.py:72]: TODO to discuss

**Comment by marielejeune** [comment]
Hi @matthieusaison , @sebastienbeau 
Since this PR seems to not evolve anymore, I've created a new one for `shopinvader_api_lead`: https://github.com/shopinvader/odoo-shopinvader/pull/1525

Mine is less complete than yours, because our customer only needs a really simple service. But the addon can then evolve if you still need it.

**Comment by sebastienbeau** [comment]
PR have been merged here https://github.com/shopinvader/odoo-shopinvader/pull/1525

**Pull request closed without merging**